MVCDemos/Views/Toolbar/Multiline.cshtml

```diff
@@ -0,0 +1,337 @@
+﻿@model DevExtreme.MVC.Demos.ViewModels.DropDownButtonViewModel
+
+@{
+    var toolbarModes = new [] { "Multiline mode", "Singleline mode" };
+    var listTypes = new [] {
+        new { icon = "orderedlist", alignment = "orderedlist", hint = "Ordered" },
+        new { icon = "bulletlist", alignment = "bulletlist", hint = "Bullet" }
+    };
+}
+
+<div>
+    @(Html.DevExtreme().Sortable()
+        .MinWidth(500)
+        .MinHeight(150)
+        .MaxHeight(370)
+        .Handles("right")
+        .Content(
+            Html.DevExtreme().Toolbar()
+                .ID("toolbar")
+                .Multiline(false)
+                .Items(items => {
+                    items
+                        .Add()
+                        .Widget(w => w
+                            .Button()
+                            .Icon("undo")
+                            .OnClick("undoButtonClick")
+                        )
+                        .Location(ToolbarItemLocation.Before);
+                    
+                    items
+                        .Add()
+                        .Widget(w => w
+                            .Button()
+                            .Icon("redo")
+                            .OnClick("redoButtonClick")
+                        )
+                        .Location(ToolbarItemLocation.Before);
+                    
+                    items
+                        .Add()
+                        .CssClass("toolbar-separator-container")
+                        .Template(new TemplateName("separator-template"))
+                        .MenuItemTemplate(new TemplateName("menu-separator-template"))
+                        .Location(ToolbarItemLocation.Before)
+                        .LocateInMenu(ToolbarItemLocateInMenuMode.Auto);
+
+                    items
+                        .Add()
+                        .Widget(w => w
+                            .DropDownButton()
+                            .DisplayExpr("Text")
+                            .KeyExpr("Size")
+                            .SelectedItemKey("14")
+                            .UseSelectMode(true)
+                            .DataSource(Model.FontSizes)
+                            .OnSelectionChanged("changeSize")
+                            .ItemTemplate(@<text><div style="font-size: <%- Text %>;"><%- Text %></div></text>)
```

Comment:
I think it's not a good to use Razor syntax directly to pass a template. Let's pass a js function everywhere

Reply:
https://github.com/DevExpress/devextreme-demos/pull/2004/commits/34ceb58c0e4b164bffa881cc28d796407bed311d